addons/website_mass_mailing/static/src/scss/website_mass_mailing.scss

```
@@ -0,0 +1,5 @@
+@include media-breakpoint-up(md) {
+    .o_subscribe_input {
+        max-width: 300px;
+    }
+}
```

Since the "component" we're referring to it's `.s_newsletter_subscribe_form` (see the data-selector you added in `snippets_templates.xml` ), I think that we should wrap this block of code using the main component class, so:

```suggestion
.s_newsletter_subscribe_form {
    @include media-breakpoint-up(md) {
        .o_subscribe_input {
            max-width: 300px;
        }
    }
}
```